axelrod/strategies/grudger.py

```diff
@@ -67,12 +67,12 @@ def strategy(self, opponent: Player) -> Action:
             self.grudge_memory = 0
             self.grudged = False
 
+        if not self.grudged and D in opponent.history[-1:]:
```

Comment:
The code and docstring didn't match.  the docstring said to play D for 10 rounds, but the code played D for 11 rounds.  I wasn't sure which to follow, so for now, I changed the code to fit the docstring.  

for a small issue, is this a correct way to raise it?